pkgs/development/python-modules/python-ptrace/default.nix

```diff
@@ -6,12 +6,12 @@
 
 buildPythonPackage rec {
   pname = "python-ptrace";
-  version = "0.9.8";
+  version = "0.9.9";
   format = "setuptools";
```

Comment:
```suggestion
  pyproject = true;
```

Reply:
Wouldn't it be better to just migrate this in one PR instead of commenting about this in every pull request?